packages/common/container-definitions/src/deltas.ts

```diff
@@ -132,15 +213,49 @@ export interface IDeltaManager<T, U> extends IEventProvider<IDeltaManagerEvents>
     submitSignal(content: any): void;
 }
 
-/** Events emitted by a Delta Queue */
+/**
+ * Events emitted by {@link IDeltaQueue}.
+ */
+/* eslint-disable @typescript-eslint/unified-signatures */
 export interface IDeltaQueueEvents<T> extends IErrorEvent {
```

Comment:
this whole interface should probably just be marked internal. these events are not for consumers

Reply:
Does that describe the IDeltaQueue interface as well, or just the events?